internal/wsproxy/recorder.go

```diff
@@ -21,6 +22,19 @@ var (
 	errAlreadyFinished     = errors.New("recording already finished")
 )
 
+var (
+	recordedSessionDuration = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "twingate_gateway",
```

Comment:
When we change the `namespace` constant in `internal/metrics.go`, how do we know to change this string here?